locales/base.json

```
@@ -36,6 +36,7 @@
     "masa": "Claim a free .celo domain, refer friends and earn rewards",
     "llamaswap": "Find the best route for your trade giving you the best execution prices in DeFi",
     "immortalx": "Trade crypto with low fees directly from your wallet, with up to 50x leverage",
-    "sarafu": "Create, use and explore Community Asset Vouchers with Grassroots Economics"
+    "sarafu": "Create, use and explore Community Asset Vouchers with Grassroots Economics",
+    "hedgey": "On-chain token vesting for your team and investors"
```

We have a few unpublished guidelines on description copy (we're working to share them). Does this phrasing work for you?

```suggestion
    "hedgey": "Empower your team and investors with on-chain token vesting"
```

---

Sounds good to me!